content/applications/finance/fiscal_localizations/argentina.rst

```diff
@@ -948,6 +948,77 @@ the corresponding layout and fiscal stamps stated in the :ref:`Invoice printed r
 .. seealso::
    :doc:`Client account creation <../../websites/ecommerce/checkout_payment_shipping/checkout>`
 
+Liquidity Product Direct Sales
+----------------------------------------
+
+The liquidity product direct sale is used, for example, in the cases where sales are made by 
+the order of a third-party.
+
+For these cases, the seller and the proprietary company of the goods, will be able to register 
+their corresponding sales and purchase. 
+
+This feature is included when installing the Argentinian EDI module (l10n_ar_edi). To install 
+or update this module, go to :guidable:’Apps > Remove filters > Search ‘l10n_ar_edi’’.
+
+.. image:: ecuador/l10n_ar_edi.png
+   :align: center 
+   :alt: l10n_ar EDI Module
+
+.. note::
+	The Argentinian EDI module is dependent on the previous installation of the Invoicing or 
+Accounting Apps and the Argentinian base module (l10n_ar).
+
+Configurations
+~~~~~~~~~~~~~~
+
+Document Types
+**************
+
+This feature includes the addition of the following document types: ‘23', '24', '25', '26', 
+'27', '28', '33', '43', '45', '46', '48', '58', '60', '61', ‘150', '151', '157', '158', '161', 
+'162', '164', '166', '167', '171', '172', '180', '182', '186', '188', '332'.
+
+It is important to note that by default these document types are inactive. For the workflow 
+to work, it is necessary to activate these documents. To find these documents, remove all filters 
+in the menu ‘Tipo de Documentos’ and activate them by toggling the option ‘Active’.
+
+.. image:: ecuador/l10n_ar_document_types.png
+   :align: center 
+   :alt: l10n_ar Document Types
+
+Purchase Journal
+****************
+
+A purchase journal is needed to generate an electronic vendor bill with a document type ‘Liquidity Product’ 
+for the order for “company A”. This journal will have to have the sync with AFIP active as this electronic 
+document will be sent to the AFIP.
```

Comment:
Hi @rps-odoo, does the user have to create this purchase journal? If so, what are the steps to do so? Please explain how to navigate within the Accounting app to create a purchase journal, and then detail the fields the user needs to complete the configuration :)

Reply:
Hi @samueljlieber this is my suggestion:

To modify the existing purchase journal, or create a new purchase journal, navigate to
:menuselection:`Accounting --> Configuration --> Journals`. Then, click the :guilabel:`New` button,
and fill in the following required information:

- :guilabel:`Type`: select :guilabel:`Purchase` from the drop-down menu for vendor bill journals.
- :guilabel:`Use Documents`: check this field for the journal to be able to generate electronic
  documents (in this case the electronic purchase invoice).
- :guilabel:`Is AFIP POS`: select :guilabel: check this field for the journal to be able to generate electronic documents (in this case liquidity product bill).
- :guilabel:`AFIP POS System`: select :guilabel:`Electroni Invoice - Web Service` from the drop-down menu in order to send the electronic document to AFIP via Web Service.
- :guilabel:`AFIP POS Number`: is the number configured in the AFIP to identify the operations related to this AFIP POS.
- guilabel:`AFIP POS Address`: is the field related to the commercial address registered for the POS, which is usually the same address as the company. For example, if a company has multiple stores (fiscal locations) then the AFIP will require the company to have one AFIP POS per location. This location will be printed in the invoice report